<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SdNdeDataPoolEditWidget</class>
 <widget class="QWidget" name="C_SdNdeDataPoolEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1000,1000,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="stw_opensyde_gui::C_SdNdeDataPoolViewWidget" name="pc_WidgetDpView" native="true"/>
     </item>
     <item>
      <widget class="stw_opensyde_gui::C_SdNdeDataPoolComListWidget" name="pc_ComListsWidget" native="true"/>
     </item>
     <item>
      <widget class="stw_opensyde_gui::C_SdNdeDataPoolListsWidget" name="pc_ListsWidget" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui::C_SdNdeDataPoolViewWidget</class>
   <extends>QWidget</extends>
   <header>C_SdNdeDataPoolViewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdNdeDataPoolListsWidget</class>
   <extends>QWidget</extends>
   <header>C_SdNdeDataPoolListsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdNdeDataPoolComListWidget</class>
   <extends>QWidget</extends>
   <header>C_SdNdeDataPoolComListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
